.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+# Related issue
+- One line for each issue. If merging this pull request should auto-close the issue on that line,
+preface the issue URL/number with "Fixes". Every pull request should have at least one issue
+linked.
+
+# Proposed changes
+- change 1
+- change 2
+
+# Checklist
+- [ ] Follow the [Google developer documentation style guide](https://developers.google.com/style)
+- [ ] Issues must be listed
+
+# Check the deployed preview
+- Each pull request is automatically staged. To review the deploy preview,
+you can either click **Details** in the deploy preview check, or you can
+visit the following link and substitute `<pull-request-number>` with the
+number for this pull request:
+```shell
```

Comment:
Put a blank line before the `shell` line.

Also all of the lines in a codeblock should be left-aligned with each other. In this case that would look like:

````
```shell
https://deploy-preview-<pull-request-number>--elated-haibt-1b47ff.netlify.com/
```
````

`https://` line is left aligned with `shell` line.

Reply:
Thanks, I have done the change which you have referred to.